### DISCUSSION TITLE
Keyball61右配置のキーボール側基盤の34番のキーが反応しない

### DISCUSSION THREAD

**Discussion by hisui-studio** [Yowkees/keyball#263] [Q&A] [answered]
Keyball61右配置のキーボール側基盤の34番のキーが反応しなくて困っています。

何度か反応していたのですが、反応しにくくなり、今では全く反応しません。

他のキーは全て反応しており、キー押下状態のキーの通電は確認できています。

ダイオードの付け替えもしてみましたが、やはり34番のKeyが反応しませんでした。

またトラックボール部品は取り付けしておりません。

どうすれば解決しますでしょうか？

ご教示頂けますと幸いです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

34とシルク刻印のあるキーが反応しないということで了解しました。
このキーだけが反応しないということはProMicro周辺の半田付けは問題なく、やはり、このキーソケットもしくは、一番近くにあるダイオードそのもの半田付けが原因と考えられます。ダイオード極性とテンプラしていないかご確認ください。

また、半田付け部品取り外し時にパッドに力がかかり、パターンが断線してしまう可能性があります。
下記画像の同色どうしが導通しているか、テスター等でご確認ください。テスターをお持ちでないようでしたら、すべての同色同士の個所をエナメル線等で接続してしまうのも手です。

お手数ですがご対応お願いいたします。

![KB61R_KEY34fix](https://github.com/Yowkees/keyball/assets/84209772/430b5138-913c-49b4-954c-563716470237)

**Comment by hisui-studio** [comment, marked as answer]
ご教示くださりありがとうございます。
赤点の部分のパッドがはずられていたことで基盤と導通できていないことが原因でした。赤点同士を銅線で繋げたところ認識が確認できましたのでやり方が合っていればキーの認識不良は解決になります。

ありがとうございます！

**Comment by Yowkees** [comment]
修理できてよかったです！
